app/components/graphPanel/graph.panel.component.html

```diff
@@ -45,18 +45,60 @@ <h4 class="modal-title">Rebase branches</h4>
       </div><!-- /.modal-content -->
     </div><!-- /.modal-dialog -->
   </div>
-  <div class="modal fade" id="authorProfileModal" tabindex="-1" role="dialog">
+  <!--TODO: redesign graph node click modal-->
```

Comment:
Dont leave TODO in the code! make issues for them and then remove them before they merged in. It is very bad practise to merge in TODO comments to your master branch.